plugins/usbpro/DmxterWidget.cpp

```diff
@@ -241,87 +235,87 @@ void DmxterWidgetImpl::HandleTodResponse(const uint8_t *data,
     OLA_WARN << "Invalid RDM response from the widget";
-    callback->Run(ola::rdm::RDM_INVALID_RESPONSE, NULL, packets);
-    delete request;
+    RunRDMCallback(callback, ola::rdm::RDM_INVALID_RESPONSE);
     return;
   }
 
-  uint8_t version = data[0];
-  uint8_t response_code = data[1];
-
-  if (version != 0) {
-    OLA_WARN << "Unknown version # in widget response: " <<
-      static_cast<int>(version);
-    callback->Run(ola::rdm::RDM_INVALID_RESPONSE, NULL, packets);
-    delete request;
+  const ResponseHeader *header = reinterpret_cast<const ResponseHeader*>(data);
+  if (header->version != 0) {
+    OLA_WARN << "Unknown version # in widget response: "
+             << static_cast<int>(header->version);
+    RunRDMCallback(callback, ola::rdm::RDM_INVALID_RESPONSE);
     return;
   }
 
-  ola::rdm::rdm_response_code code = ola::rdm::RDM_COMPLETED_OK;
-  switch (response_code) {
+  ola::rdm::RDMStatusCode status_code = ola::rdm::RDM_COMPLETED_OK;
+  switch (header->response_code) {
     case RC_CHECKSUM_ERROR:
-      code = ola::rdm::RDM_CHECKSUM_INCORRECT;
+      status_code = ola::rdm::RDM_CHECKSUM_INCORRECT;
       break;
     case RC_FRAMING_ERROR:
     case RC_FRAMING_ERROR2:
     case RC_BAD_STARTCODE:
-      code = ola::rdm::RDM_INVALID_RESPONSE;
+      status_code = ola::rdm::RDM_INVALID_RESPONSE;
       break;
     case RC_BAD_SUB_STARTCODE:
-      code = ola::rdm::RDM_WRONG_SUB_START_CODE;
+      status_code = ola::rdm::RDM_WRONG_SUB_START_CODE;
       break;
     case RC_WRONG_PDL:
     case RC_BAD_PDL:
-      code = ola::rdm::RDM_INVALID_RESPONSE;
+      status_code = ola::rdm::RDM_INVALID_RESPONSE;
       break;
     case RC_PACKET_TOO_SHORT:
-      code = ola::rdm::RDM_PACKET_TOO_SHORT;
+      status_code = ola::rdm::RDM_PACKET_TOO_SHORT;
       break;
     case RC_PACKET_TOO_LONG:
-      code = ola::rdm::RDM_INVALID_RESPONSE;
+      status_code = ola::rdm::RDM_INVALID_RESPONSE;
       break;
     case RC_PHYSICAL_LENGTH_MISMATCH:
-      code = ola::rdm::RDM_PACKET_LENGTH_MISMATCH;
+      status_code = ola::rdm::RDM_PACKET_LENGTH_MISMATCH;
       break;
     case RC_PDL_LENGTH_MISMATCH:
-      code = ola::rdm::RDM_PARAM_LENGTH_MISMATCH;
+      status_code = ola::rdm::RDM_PARAM_LENGTH_MISMATCH;
       break;
     case RC_TRANSACTION_MISMATCH:
-      code = ola::rdm::RDM_TRANSACTION_MISMATCH;
+      status_code = ola::rdm::RDM_TRANSACTION_MISMATCH;
       break;
     case RC_BAD_RESPONSE_TYPE:
-      code = ola::rdm::RDM_INVALID_RESPONSE_TYPE;
+      status_code = ola::rdm::RDM_INVALID_RESPONSE_TYPE;
       break;
     case RC_GOOD_LEVEL:
       if (waiting_for_dub_response) {
-        code = ola::rdm::RDM_DUB_RESPONSE;
+        status_code = ola::rdm::RDM_DUB_RESPONSE;
       } else {
-        OLA_INFO << "Got response code " << static_cast<int>(response_code);
-        code = ola::rdm::RDM_INVALID_RESPONSE;
+        OLA_INFO << "Got response status_code "
+                 << static_cast<int>(header->response_code);
```

Comment:
Fixed. On the DMXter side these are called response codes. It was a bad search and replace.